doc/modules/reference.rst

```diff
@@ -115,7 +137,8 @@ uses.
    reorder_img
    smooth_img
    swap_img_hemispheres
-
+   coord_transform
+   get_bounds
```

Comment:
> Ok, what's the nilearn policy for exposing / not-exposing functions ?

There is no 'policy', I am afraid. It's more a question of 'design'. The
way I think of it is that nilearn needs to be as accessible as possible
(my visit to Oxford convinced me even more of this) and for this we need
to expose only high-level functionnality that is understandable and
useable to end users doing neuroimaging.